samples/sampler/stories/qa/Panel.js

```diff
@@ -0,0 +1,84 @@
+import Button from '@enact/sandstone/Button';
```

Comment:
I think `Panels.js` is proper than `Panel.js`.

and need "noPanels" option

	props: {
		noPanels: true
	}

Reply:
It's moved.

Reply:
I'll add 
props: {
	noPanels: true
}